### ISSUE TITLE
Add Sample Mixed Lua/Objective-C Extension to Contribution Guide 

### ISSUE THREAD

**Issue by latenitefilms** [Hammerspoon/hammerspoon#1345]
It would be great if someone ( @cmsj or @asmagill ?) could add a very bare, plain, simple and well documented Mixed Lua/Objective-C Extension to the Contribution Guide.

**Comment by latenitefilms** [comment]
Ideally something "self contained" with it own Makefile.

**Comment by asmagill** [comment]
It's been a while since I've reviewed this and made sure it's current, but that was the intention behind https://github.com/asmagill/HS_ModuleWalkthrough

**Comment by latenitefilms** [comment]
This is perfect - thanks @asmagill !

Maybe this should be added to `README.md`?

**Comment by cmsj** [comment]
@asmagill maybe the module walkthrough could live in our Wiki and be linked from README.md?

**Comment by latenitefilms** [comment]
@asmagill - Should the Module Walkthrough be added to the Wiki?

**Comment by asmagill** [comment]
Absolutely, no question in my mind... but my current partial one is so out of date its probably worse than just looking at our newer modules for examples.  I am expecting to have some time later in October to do a number of write-ups and this is one of them I have in mind.
